src/modules/Credential/containers/IssuedList/useIssuedList.tsx

```diff
@@ -1,28 +1,48 @@
 import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { useLoaderData } from 'react-router-dom';
-import { CredentialsRes, CredentialStatus, deleteCredentialAdaptor, revokeCredentialAdaptor } from 'src/core/adaptors';
+import { useLoaderData, useNavigate } from 'react-router-dom';
+import { config } from 'src/config';
+import {
+  CredentialsRes,
+  CredentialStatus,
+  deleteCredentialAdaptor,
+  getCredentialsAdaptor,
+  revokeCredentialAdaptor,
+} from 'src/core/adaptors';
 import { StatusProps } from 'src/modules/General/components/Status/index.types';
 
 export const useIssuedList = () => {
   const { t: translate } = useTranslation();
+  const navigate = useNavigate();
   const { credentialList } = (useLoaderData() as { credentialList: CredentialsRes }) || {};
-  const { items: list } = credentialList || [];
-  const [currentList, setCurrentList] = useState(list);
+  const [currentCredentialList, setCurrentCredentialList] = useState(credentialList);
+  const currentList = currentCredentialList?.items || [];
   const [selectedCredential, setSelectedCredential] = useState('');
-  const [openModal, setOpenModal] = useState<{ name: 'revoke' | 'delete' | ''; open: boolean }>({
+  const [openModal, setOpenModal] = useState<{ name: 'revoke' | 'delete' | 'copy' | ''; open: boolean }>({
     name: '',
     open: false,
   });
+  const [url, setUrl] = useState('');
   const [page, setPage] = useState(1);
-  const totalPage = 1;
+  const totalPage = Math.ceil((currentCredentialList?.total || 1) / (currentCredentialList?.limit || 10));
 
-  const status: Record<Exclude<CredentialStatus, 'ISSUED'>, StatusProps> = {
+  const status: Record<CredentialStatus, StatusProps> = {
+    ISSUED: { label: 'Issued', theme: 'warning' },
     PENDING: { label: translate('credential-status.pending'), theme: 'warning' },
     ACTIVE: { label: translate('credential-status.active'), theme: 'success' },
     REVOKED: { label: translate('credential-status.revoked'), theme: 'error' },
   };
 
+  const onChangePage = async (newPage: number) => {
+    setPage(newPage);
+    const { data, error } = await getCredentialsAdaptor(newPage);
+    if (error) {
+      console.log(error);
+      return;
+    }
+    data && setCurrentCredentialList(data);
+  };
+
```

Comment:
no need to log error. just this line is enough:
   data && setCurrentCredentialList(data);